README.md

```diff
@@ -1,4 +1,16 @@
 wp-cli/ideas
 ============
 
-Have an idea for a WP-CLI command? [Create a new issue](https://github.com/wp-cli/ideas/issues/new) or [up-vote an existing issue](https://github.com/wp-cli/ideas/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).
+### Have an idea for a WP-CLI command?**
+
+Feel free to [up-vote an existing issue](https://github.com/wp-cli/ideas/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc) if you can find one that matches, or [create a new issue](https://github.com/wp-cli/ideas/issues/new) if no one has proposed the idea yet.
+
+### Curious as to how ideas become WP-CLI commands?
+
+The process generally works like this:
+
+0. **New issue is created.** Good issues include substantial detail into what the command might do, how it might work, and what value it would add to the ecosystem.
+1. **Some discussion takes place.** Issues are evaluated against [WP-CLI's philosophy](https://make.wordpress.org/cli/handbook/philosophy/), technical feasibility, and estimated impact. Everyone is free to weigh in; we value thoughtful consideration that introduces new viewpoints.
```

Comment:
Maybe add something like "It's here that the decision is made to either add a new command or add a new subcommand to an existing command, or add the functionality as a new option to an existing command or subcommand."